### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,15 +16,15 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "^1.0@dev",
-        "php-http/discovery": "^0.4",
+        "php-http/httplug": "^1.0-beta",
+        "php-http/discovery": "^0.5",
```
Comment (ddeboer) [inline composer.json]: @sagikazarmark This gets somewhat tiresome. Would `~0.5` (so including `0.6` and up) be okay with you?

**Comment by ddeboer** [comment]
I don’t quite understand why Scrutinizer fails on the Composer dependencies when they are fine on Travis.

File: composer.json (hunk 1 above)
Comment (sagikazarmark) [inline composer.json]: :+1:

Comment (sagikazarmark) [inline composer.json]: Isn't it 1.0.0-beta? This might be the reason for failing as travis caches dependencies.

**Comment by ddeboer** [comment]
Scrutinizer still failing on 1.0.0-beta. Could it have something to do with their outdated Composer version?

Anyway, let’s just merge this PR.

**Comment by sagikazarmark** [comment]
I can't think of any features we rely on that has been added recently. Let's ignore it for now.

**Comment by ddeboer** [comment]
Let’s merge? :wink:

**Comment by ddeboer** [comment]
Thanks!

**Comment by Nyholm** [comment]
Awesome!

**Pull request merged**